test/testdata/notary.pem

```diff
@@ -0,0 +1,48 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
That's also my concern. If I make each test generate cert/key, we have to resign the test images. I'm considering this option:
1. for this PR, each test will fetch cert from a pre-created AKV with imported cert/key. So we won't create new key vaults in the test but the integration with AKV is still tested.
2. I'll have another PR that will build/sign test images and upload to newly-created ACR for each test so that we don't need to pull from wabbitnetworks registry all the time. At the same time, we could have the AKV to generate new cert/key instead of using the same cert always.